src/Gblix/Controllers/ApiTraits/Create.php

```diff
@@ -57,23 +56,31 @@ final protected function makeStore(Request $request, $job, RepositoryInterface $
 
         $user = $request->user();
 
-        if (!$job instanceof Action) {
```

Comment:
Aqui poderia ser usado um method_exists pra make, não? 
Ou alguma interface para padronizar a operação quando não for Action..